style.css

```diff
@@ -277,3 +277,82 @@ body {
   justify-content: space-between;
   padding-right: 60px;
 }
+
+footer {
+  background-color: #6070ff;
+  background-image: url(pimages/footer.png);
+  background-repeat: no-repeat;
+  background-size: 70% 90%;
+  background-position: right;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 100px 0 0 0;
+  padding: 6rem 1rem;
+  gap: 2rem;
+  margin-top: 1rem;
+  height: 100%;
+}
+
+.lasting {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.lasting h3 {
+  font-size: 40px;
+  font-weight: 700;
+  color: #fff;
+  text-align: center;
+}
+
+.lasting p {
+  font-size: 20px;
+  font-weight: 400;
+  color: #ebebff;
+  text-align: center;
+  padding-left: 30px;
+  padding-right: 30px;
+}
+
+.form ul {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.form input,
+textarea {
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 1rem;
+  width: 25em;
+  color: #b3bac5;
+}
+
+.footer-btn {
+  background: #fff;
+  border-radius: 8px;
+  width: 142px;
+  height: 48px;
+  text-align: center;
+  padding: 12px 16px;
+  color: #6070ff;
+  font-size: 17px;
+  font-weight: 500;
+  border: none;
+  transition-duration: 0.4s;
+  cursor: pointer;
+}
+
+.footer-btn:hover {
+  background-color: red;
+  color: #fff;
+}
+
+.footer-btn:active {
+  background-color: black;
+  color: #fff;
+}
```

Comment:
- [ ] As suggested by the previous reviewer, please make sure to implement the button interactions according to the Figma template 📌 Please consider implementing the following for `hover` and `active` states 👇✨

```
.footer-btn:hover {
  background: #6070FF;
  color: #fff;
  box-shadow: 0px 8px 16px rgba(64, 83, 252, 0.24);
}

.footer-btn:active {
  background: #2230D2;
  color: #fff;
}
```